react-stopwatch-component/react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: 0,
+      active: false
+    };
+    this.formatTime = this.formatTime.bind(this);
+    this.start = this.start.bind(this);
+    this.stop = this.stop.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+  start() {
+    this.timerId = setInterval(
+      () => this.tick(), 1000
+    );
+    this.setState({
+      active: true
+    });
+  }
+
+  stop() {
+    clearInterval(this.timerId);
+    this.setState({
+      active: false
+    });
+  }
+
+  reset() {
+    this.setState({
+      time: 0
+    });
+  }
+
+  tick() {
+    this.setState({
+      time: this.state.time + 1
+    });
+  }
+
+  formatTime(time) {
+    if (time < 10) {
+      return `0${time}`;
+    }
+    return time;
+  }
```

Comment:
Minor suggestion, but the time formatting function should consistently return a string.